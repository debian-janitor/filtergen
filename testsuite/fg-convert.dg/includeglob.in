input eth0 {
    proto tcp dport ssh accept;
    include testsuite/fg-convert.dg/includeglob.d/* ;
};
