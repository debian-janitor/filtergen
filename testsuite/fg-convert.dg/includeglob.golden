( siblist
  ( input eth0
   )
  child
  ( siblist
    ( siblist
      ( proto tcp
       )
      child
      ( siblist
        ( dport ssh
         )
        child
        ( accept
         )
       )
      next
      ( siblist
        ( proto tcp
         )
        child
        ( siblist
          ( dport 1
           )
          child
          ( accept
           )
         )
        next
        ( siblist
          ( proto tcp
           )
          child
          ( siblist
            ( dport 2
             )
            child
            ( accept
             )
           )
          next
          ( siblist
            ( proto tcp
             )
            child
            ( siblist
              ( dport 3
               )
              child
              ( accept
               )
             )
           )
         )
       )
     )
   )
 )
